Update ug for v1.3 (#168)
AY2324S1-CS2103T-T09-1/tp: docs/UserGuide.md

---
  layout: default.md
  title: "User Guide"

---

# JobFestGo User Guide

**Job Fest event planning has never been easier!**

JobFestGo is a **desktop application** built for job event planners to manage contacts and tasks for their events.

Here’s an **overview** of how JobFestGo can help you with your event planning:
- Store information about your events and their associated contacts
- Track status of your event tasks

> - Keep track of upcoming deadlines

Reviewer: ```suggestion
- Keep track of upcoming task deadlines
```

Author: added

* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
</box>

## Adding a contact : `add_contact`

Adds a contact to JobFestGo.

> 

Reviewer: I don't think we have to include the `A contact must have mandatory fields name, phone number, email and address`, since in the Features section, we mentioned that the fields without `[]` are mandatory (in line 117)

Author: removed

* A contact cannot be added if their phone number already exists
* A contact cannot be added if their name already exists

<box type="warning" seamless>

> **WARNING** : Please ensure there is no more than **one** whitespace in-between each word for the contact name!

Reviewer: ```suggestion
**WARNING** : Please ensure there is no more than **one** whitespace in-between each word for `NAME`!
```

Author: added

> 

Reviewer: Maybe can also include that the `name ` should only consists of alphanumeric characters (either as a warning or include it in the notes)

Author: added

Format: `add_event n/NAME d/DATE a/ADDRESS`

> * An event must have a mandatory name, date and address.

Reviewer: Likewise for this point too :)

Author: removed

Format: `add_task td/TASK_DESCRIPTION d/DEADLINE ev/EVENT_NAME`

* A task must have a mandatory task description, deadline and event name.
* Task Description can take any alphanumeric value, alongside hyphen, comma, brackets and full stops.
* Deadline is a date in the format YYYY-MM-DD.
* Deadline should not be before today's date.
* Event name should be the **name of an already existing event**.

> * Reminder will be shown on the home page of JobFestGo for tasks due less than 4 days from the deadline.

Reviewer: ```suggestion
* Reminder will be shown on the home page of JobFestGo for tasks that are due in 4 days from the deadline.
```
